### ISSUE TITLE
Is it possible to render a head model with real texture?

### ISSUE THREAD

**Issue by TonyLiangJunsheng** [CaffeineViking/vkhr#50]

**Comment by CaffeineViking** [comment]
Thanks for opening an issue on this Tony! Indeed, it is something that's missing and something I have been meaning to add eventually to polish the demo. I just never got around to it. Most of the code is already written for it, assuming we are using OBJ/MTL, all that needs doing is uploading the textures to the GPU and then reading from the correct set of UVs. The other benefit besides "polish", is that if the scalp is colored with the same color as the hair, areas where hair is missing won't be as noticeable. It's a common trick used in the game industry when rendering hair. I'll see if I can find some time to do this.

**Comment by TonyLiangJunsheng** [comment]
Yes thanks for your response. I saw some code associated with it. And I also have tried to add this function based on it. But I need to understand the whole structure. It is a huge job, but I will do it in my free time :)
